x-pack/legacy/plugins/ml/public/application/components/ml_in_memory_table/types.ts

```diff
@@ -28,6 +28,7 @@ export interface FieldDataColumnType<T> {
   render?: RenderFunc;
   footer?: string | ReactElement | FooterFunc;
   textOnly?: boolean;
+  scope?: 'col' | 'row' | 'colgroup' | 'rowgroup';
```

Comment:
is it possible to import this type from eui?

Reply:
Looks so, it is `TableHeaderCellScope`

Reply:
Decided to copy those `scope` values locally for now. We plan to switch over to using the types which are now available in EUI, hopefully for 7.7.